src/modules/inspect-correlation/inspect-correlation.ts

```diff
@@ -91,9 +109,36 @@ export class InspectCorrelation {
 
   public resizeInspectPanel(mouseEvent: MouseEvent): void {
     const mouseYPosition: number = mouseEvent.clientY;
-    const inspectPanelHeightWithStatusBar: number = this.bottomPanelResizeDiv.parentElement.parentElement.clientHeight + this._menuBarHeight;
+
+    const menuBarHeight: number = 40;
+    const inspectCorrelation: HTMLElement = this.bottomPanelResizeDiv.parentElement.parentElement;
+    const inspectPanelHeightWithStatusBar: number = inspectCorrelation.clientHeight + menuBarHeight;
+
+    const minInspectPanelHeight: number = 250;
+
     const newBottomPanelHeight: number = inspectPanelHeightWithStatusBar - mouseYPosition;
 
-    this.bottomPanelHeight = Math.max(newBottomPanelHeight, this._minInspectPanelHeight);
+    this.bottomPanelHeight = Math.max(newBottomPanelHeight, minInspectPanelHeight);
+  }
+
+  public resizeTokenViewer(mouseEvent: MouseEvent): void {
```

Comment:
Does this need be public?

```suggestion
  private _resizeTokenViewer(mouseEvent: MouseEvent): void {
```